client/client.go

```diff
@@ -30,7 +30,7 @@ type Config struct {
 
 type Client interface {
 	Ping(ctx context.Context) (*rpcpb.PingResponse, error)
-	Start(ctx context.Context, execPath string, opts ...OpOption) (*rpcpb.StartResponse, error)
+	Start(ctx context.Context, execPath string, numNodes uint32, opts ...OpOption) (*rpcpb.StartResponse, error)
```

Comment:
On second thought, can we make `numNodes` as `OpOption`?

`WithNumNodes(n uint32)` can set a non-default number of node?

Reply:
That way, we. don't need to break the client interface